ext/cl_khr_device_uuid.asciidoc

```diff
@@ -0,0 +1,92 @@
+// Copyright 2018-2020 The Khronos Group. This work is licensed under a
+// Creative Commons Attribution 4.0 International License; see
+// http://creativecommons.org/licenses/by/4.0/
+
+[[cl_khr_device_uuid]]
+== Unique Device Identifiers
+
+This section describes the *cl_khr_device_uuid* extension.
+
+This extension adds the ability to query a universally unique identifier
+(UUID) for an OpenCL driver and OpenCL device.
+The UUIDs returned by the query may be used to identify drivers and devices
+across processes or APIs.
+
+=== General information
+
+==== Version history
+
+[cols="1,1,3",options="header",]
+|====
+| *Date*     | *Version* | *Description*
+| 2020-08-27 | 1.0.0     | First assigned version.
+|====
+
+// == New API Enums
+// 
+// Accepted value for the _param_name_ parameter to *clGetDeviceInfo*:
+// 
+// [source,c]
+// ----
+// #define CL_DEVICE_UUID_KHR          0x106A
+// #define CL_DRIVER_UUID_KHR          0x106B
+// #define CL_DEVICE_LUID_VALID_KHR    0x106C
+// #define CL_DEVICE_LUID_KHR          0x106D
+// #define CL_DEVICE_NODE_MASK_KHR     0x106E
+// ----
+// 
+// Constants describing the size of the driver and device UUIDs, and the device LUID:
+// 
+// [source,c]
+// ----
+// #define CL_UUID_SIZE_KHR            16
+// #define CL_LUID_SIZE_KHR            8
```

Comment:
These constants aren't in the XML so the headers will be the only written source for their values (once they're added there).

The `cl_khr_device_uuid` entry could have a `<require comment="Constants">` section.

Reply:
The XML change is pretty straightforward - I'll add that and include it in this PR.

Do we want to describe the **values** of these constants here in the spec though, or only in the XML and headers?  We do include the values of several other constants in the spec, but they're mostly numerical limits in the half-precision extension.  We don't include the values of other enumerants (though we have talked about adding them).